packages/devtools_app_shared/lib/src/ui/flex_split_column.dart

```diff
@@ -2,25 +2,28 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
 
+/// @docImport 'package:devtools_app_shared/src/ui/common.dart';
+library;
+
 import 'package:flutter/material.dart';
 
 import 'split_pane.dart';
 
-/// A widget that takes a list of [children] and lays them out in a column where
+/// A widget that takes a list of `children` and lays them out in a column where
 /// each child has a flexible height.
 ///
-/// Each child in [children] must have an accompanying header in [header]. Since
-/// each header must be a [PreferredSizeWidget], it is common to use the shared
-/// widgets [AreaPaneHeader] or [BlankHeader] for the column [headers].
+/// Each child in `children` must have an accompanying header in [headers].
```

Comment:
is there a lint to catch these types of errors?

Reply:
There is; the `comment_references` lint. I hope to turn it on after fixing these.